src/main/java/org/folio/circulation/infrastructure/storage/inventory/HoldingsRepository.java

```diff
@@ -26,10 +28,14 @@ public HoldingsRepository(CollectionResourceClient holdingsClient) {
   CompletableFuture<Result<Holdings>> fetchById(String id) {
     final var mapper = new HoldingsMapper();
 
+    return fetchAsJson(id)
+      .thenApply(mapResult(mapper::toDomain));
+  }
+
+  public CompletableFuture<Result<JsonObject>> fetchAsJson(String id) {
```

Comment:
Why do we need to extract this method? And if we do, should we name it fetchHoldingsRecordAsJson to match  8  
ItemRepository.fetchItemAsJson() ?